test/Microsoft.ML.CodeGenerator.Tests/Microsoft.ML.CodeGenerator.Tests.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
```

Comment:
You don't need this. It is automatic from 

[https://github.com/dotnet/machinelearning/blob/master/test/Directory.Build.props#L9](https://github.com/dotnet/machinelearning/blob/master/test/Directory.Build.props#L9)